### PR TITLE
Viya 3.3 dev

### PR DESCRIPTION
**Pull request by screnshaw**
merging 3.3-dev nic and task cleanup.

### CODE REVIEW

File: playbooks/pre-install-playbook/roles/virk.preinstall/defaults/main.yml [hunk 1]
```diff
@@ -141,12 +141,17 @@ sas_yum_urls:
...
+#max_num_nics: 1             ## maximum number of network interfaces (excluding loopback)
 ## the command used to count active NICs
 #nic_count_command: "ls -A /sys/class/net | grep -v lo | wc -l"
 #nic_count_command: "netstat -i|grep -ve ^lo -e ^Iface -e ^Kernel | wc -l"
-nic_count_command: "ip link show up | grep -ve link -e ': lo:' | wc -l"
+#nic_count_command: "ip link show up | grep -ve link -e ': lo:' | wc -l"
+## we should be counting IPs, and not NICs.
+##
+max_num_priv_ips: 1             ## maximum number of network interfaces
+#priv_ip_count_command: "ip -4 -o addr | awk '!/^[0-9]*: ?lo|link\\/ether/ {gsub(\"/\", \" \"); print $4}' | wc -l"
```
Comment (kevinlinglesas) [inline playbooks/pre-install-playbook/roles/virk.preinstall/defaults/main.yml]: Why include this line?  Seems you have revised the implementation of the command in the next line.

Comment (erwangranger) [reply playbooks/pre-install-playbook/roles/virk.preinstall/defaults/main.yml]: No particular reason. I commented it out when I switched to the new implementation, and forgot to remove it after I confirmed the new method worked.

File: playbooks/pre-install-playbook/roles/virk.preinstall/tasks/pre.ssh_maxstartups_config.yml [hunk 2]
```diff
@@ -59,7 +68,8 @@
         backup=yes
       register: sshd_config
 
-    - debug: var=sshd_config
+    # - name: Display the result of the update of the sshd_config file
```
[the remark this answers is not in the document]
Comment (erwangranger) [reply playbooks/pre-install-playbook/roles/virk.preinstall/tasks/pre.ssh_maxstartups_config.yml]: I think that after giving the task a proper name, I decided this debug message wasn't that useful. I forget exactly what it looked like. We could probably delete it altogether.

**Comment by thpang** [comment]
@screnshaw was this reviewed? Wondering if we can moved forward with this merge request? Thanks.

**Comment by screnshaw** [comment]
Yes. There were verified, reviewed, and merged into 3.3-dev, but needs to be merged into the main 3.3 branch.

**Pull request merged**